portfolio/src/main/webapp/script.js

```diff
@@ -12,36 +12,59 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function getLogin() {
+  fetch('/login').then(response => response.json()).then((account) => {
+    console.log(account);
+    loginLink = document.getElementById("login-link");
+    //If not logged in
+    if(account.loggedIn == false) {
+      loginLink.href = account.loginUrl;
+      loginLink.innerText = "Login";
+      document.getElementById("reminder-container").style.display ="block";
+    }
+    //If logged in
+    else {
+      loginLink.href = account.loginUrl;
+      loginLink.innerText = "Logout";
+      document.getElementById("name").value = account.nickname;
+      document.getElementById("email").value = account.email;
+      document.getElementById("h2-container").style.display ="block";
+      document.getElementById("h2-container").innerText ="Welcome " + account.nickname;
+      document.getElementById("form-container").style.display ="block";
+    }
+    console.log(account.email);
+  });
+}
 
 /* Gets past comment history from the DataServlet server and adds it to the DOM */
 function getCommentHistory() {
-    // Show the title
-    const hiddenElement = document.getElementById('hidden');
-    hiddenElement.style.display = 'block';
+  // Show the title
+  const hiddenElement = document.getElementById('hidden');
+  hiddenElement.style.display = 'block';
 
-    // Fetch the comments
-    const commentNumberElement = document.getElementById('comment-number');
-    const sortParamElement = document.getElementById('sort-param');
+  // Fetch the comments
+  const commentNumberElement = document.getElementById('comment-number');
+  const sortParamElement = document.getElementById('sort-param');
 
-    fetch('/history?comment-number=' + commentNumberElement.value + '&sort-param=' + sortParamElement.value).then(response => response.json()).then((comments) => {
-    const historyElement = document.getElementById('history-container');
+  fetch('/history?comment-number=' + commentNumberElement.value + '&sort-param=' + sortParamElement.value).then(response => response.json()).then((comments) => {
+  const historyElement = document.getElementById('history-container');
```

Comment:
need 2 more space for indentation from this line till the close of the { for the above fetch line.

Consider making line 49 shorter so that it is visible without the need to scroll and less likely for developers to miss the "{" at the end